Mention when this mode is useful, and when it is not.

tarsius opened the pull request:
Also drop support for older Magit releases. That way we no longer have to mention that it requires `file-notify-support` and therefore Emacs v24.4. The current Magit release depends on that anyway.

tarsius committed:
Mention when this mode is useful, and when it is not.

ruediger commented:
Is support for older releases of magit broken? If not then I don't see a reason to drop support.

I also don't agree with the description of "usefulness". The package updates the status buffer on changes to content of a git repo. This can be useful independent of whether someone uses magit commands or external commands. The point is to avoid having to press `g`.

tarsius commented:
> Is support for older releases of magit broken? If not then I don't see a reason to drop support.

Okay, I have removed that commit.

> The package updates the status buffer on changes to content of a git repo. This can be useful independent of whether someone uses magit commands or external commands.

But Magit already does that without this package.

Magit used to only do it when the current buffer is a buffer whose major-mode derives from `magit-mode`, but I have changed that in 14a4ef60e01a33988ca2ebe046742f92f5c20c2c.

By default Magit also does not refresh the status buffer whenever a buffer is saved to a file located inside the repository, but I have added a function to do that in 6d0851e73960a60d816bfad9bf036bc62229e399.

After these changes and adding `magit-refresh-status` to `after-save-hook`, when is `magit-filenotify` still needed when just using Magit and Emacs to interact with Git repositories?

tarsius commented:
So, what do you think?

ruediger commented:
Since the change is quite new to magit (is it in any released version?), I think the README change should reflect that.

tarsius commented:
Okay, I have updated the readme.

tarsius commented:
Thanks
